python/cudf_polars/cudf_polars/dsl/utils/windows.py

```diff
@@ -88,22 +86,10 @@ def duration_to_scalar(dtype: plc.DataType, value: int) -> pa.Scalar:
     value
         The raw value as in integer. If `dtype` represents a timestamp
         type, this should be in nanoseconds.
-    months
-        Number of months
-    weeks
-        Number of weeks
-    days
-        Number of days
-    nanoseconds
-        Number of nanoseconds
-    parsed_int
-        Is this actually a representation of an integer, not a duration?
-    negative
-        Is this a negative duration?
```

Comment:
Did we forget to remove this in an earlier refactor?

Reply:
It appears so. Though it was safe to remove since they're not in the signature anymore